htdocs/install/mysql/tables/llx_entrepot.key.sql

```diff
@@ -17,4 +17,4 @@
 -- ============================================================================
 
 
-ALTER TABLE llx_entrepot ADD UNIQUE INDEX uk_entrepot_label (ref, entity);
+ALTER TABLE llx_entrepot ADD UNIQUE INDEX uk_entrepot_label (ref, entity,fk_parent);
```

Comment:
@homer8173 
remove and recreate Unique Index in migration script

Reply:
Don't worry, it's fixed for me
but this fix should also improve Dolibarr

Reply:
Sorry, you want me to report it in migration script also ?

Reply:
@homer8173 
if you change only in "/mysql/tables/", this fix is only for fresh install !
add this in "/mysql/migration/7.0.0-8.0.0.sql" for dolibarr update : 

ALTER TABLE llx_entrepot DROP INDEX uk_entrepot_label;
ALTER TABLE llx_entrepot ADD UNIQUE INDEX uk_entrepot_ref_fk_parent (ref, entity, fk_parent);

Reply:
@hregis thanks for advices !